test/services/listCourtCases/filterByReasonAndResolutionStatus.integration.test.ts

```diff
@@ -341,6 +341,152 @@ describe("Filter cases by resolution status", () => {
           "Exceptions Unresolved/Bails Trigger Unresolved"
         ]
       },
+      {
+        description:
+          "Should see cases with unresolved triggers when user is a trigger handler and searches for TRPR0010",
+        filters: {
+          reasonCodes: ["TRPR0010"]
+        },
+        user: triggerHandler,
+        expectedCases: [
+          "Exceptions Unresolved/Bails Trigger Unresolved", // Sees this case as it has the reason code
+          "No exceptions/Bails Trigger Unresolved"
+        ]
+      },
+      {
+        description:
+          "Should see cases with unresolved exception when user is a exception handler and searches for TRPR0010",
```

Comment:
Should the title have HO100300 instead of trigger code?

Reply:
🤦‍♂️